app/Http/Controllers/Api/ApiController.php

```diff
@@ -64,11 +64,12 @@ public function item($data, $code = 200, $meta = [], $transformer = null)
      * @param  array  $meta
      * @return \Illuminate\Http\JsonResponse
      */
-    public function transform($data, $code = 200, $meta = [], $include = null)
+    public function transform($data, $code = 200, $meta = [], $include = null, $endpoint = null)
```

Comment:
@chloealee This `$endpoint` parameter and the lines below referencing it can be removed now too!